examples/chat/src/main.rs

```diff
@@ -26,8 +26,8 @@ use tracing_subscriber::{layer::SubscriberExt, util::SubscriberInitExt};
 
 // Our shared state
 struct AppState {
-    user_set: Mutex<HashSet<String>>,
-    tx: broadcast::Sender<String>,
+    user_set: Mutex<HashSet<String>>, // Currently taken user names (must be unique)
+    tx: broadcast::Sender<String>, // Receivers can be created via tx.subscribe()
```

Comment:
```suggestion
    // We require unique usernames. This tracks which usernames have been taken.
    user_set: Mutex<HashSet<String>>,
    // Channel used to send messages to all connected clients.
    tx: broadcast::Sender<String>,
```

Reply:
Oh, sure. I must admit that I found the full stop annoying ;), but I can see that consistency is most important. Will change that again.